qiskit/transpiler/passes/synthesis/unitary_synthesis.py

```diff
@@ -53,12 +53,31 @@ def _choose_euler_basis(basis_gates):
     basis_set = set(basis_gates or [])
 
     for basis, gates in one_qubit_decompose.ONE_QUBIT_EULER_BASIS_GATES.items():
+
         if set(gates).issubset(basis_set):
             return basis
 
     return None
 
 
+def _choose_basis(basis_gates, basis_dict=None):
+    """Find all matching basis in the specified basis gates."""
```

Comment:
basis -> bases?  I also don't understand the sentence, though.

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/6124/commits/5d8539ac37300473270b470574aa8879c975e2f9